tracer/build/_build/Build.Steps.cs

```diff
@@ -586,9 +588,33 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
             CompressZip(SymbolsDirectory, WindowsSymbolsZip, fileMode: FileMode.Create);
         });
 
+    Target ExtractDebugInfoAndStripSymbols => _ => _
+        .After(PublishProfilerLinux, PublishNativeLoaderLinux, PublishNativeProfilerLinux)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            var directory = new DirectoryInfo(MonitoringHomeDirectory);
+            var files = directory.GetFiles("*.so");
+
+            var symbolsDirectory = SharedDirectory / "symbols";
```

Comment:
We already have the `SymbolsDirectory`, seems like a good place for them (currently used by the Windows build IIRC)? 🙂

Reply:
sure, I did not know.